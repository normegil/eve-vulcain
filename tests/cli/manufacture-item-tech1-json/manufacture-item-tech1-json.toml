bin.name = "eve-vulcain"
args = "manufacture item \"Drone Link Augmentor I\" --strict --json -vvv"
status.code = 0
