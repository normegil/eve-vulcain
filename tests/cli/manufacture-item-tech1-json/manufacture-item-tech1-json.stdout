{"searched_item_name":"Drone Link Augmentor I","markets":[{"name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","regional_average_volumes":1331,"lowest_price":185000.0}],"invention":null,"facilities":[{"facility_name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","costs":{"inputs":{"inputs":[{"name":"Tritanium","orders":[{"quantity":5556,"price_per_unit":4.12,"total":22890.72}]},{"name":"Pyerite","orders":[{"quantity":2222,"price_per_unit":10.0,"total":22220.0}]},{"name":"Mexallon","orders":[{"quantity":61,"price_per_unit":54.07,"total":3298.27}]},{"name":"Isogen","orders":[{"quantity":2,"price_per_unit":458.35,"total":916.7}]},{"name":"Nocxium","orders":[{"quantity":2,"price_per_unit":1191.13,"total":2382.26}]},{"name":"Zydrine","orders":[{"quantity":112,"price_per_unit":1845.41,"total":206685.92}]},{"name":"Megacyte","orders":[{"quantity":22,"price_per_unit":5932.53,"total":130515.65999999999}]}],"total":388909.52999999997},"job_cost":{"estimated_item_value":177913.0327276708,"system_cost_index":0.2088,"facility_tax":0.0025,"total":37859.89336444835,"job_cost_modifier":null},"total_run":426769.4233644483,"total_per_unit":426769.4233644483},"time":{"base_time":2700,"job_duration_modifier":null,"industry_skill_level":5,"advanced_industry_skill_level":5,"total_time_run":1836,"number_of_run_per_hour":1.9607843137254901,"number_of_run_per_day":47.05882352941176},"volume":{"per_unit_volume":25.0,"per_run_volume":25.0,"per_hour_volume":49.01960784313725,"per_day_volume":1176.4705882352941},"profit":{"markets_profit":[{"name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","profit_per_hour":-474057.69287146727,"profit_per_day":-11377384.628915215}]}},{"facility_name":"Maurasi - BPs Center, read BIO","costs":{"inputs":{"inputs":[{"name":"Tritanium","orders":[{"quantity":5500,"price_per_unit":4.12,"total":22660.0}]},{"name":"Pyerite","orders":[{"quantity":2200,"price_per_unit":10.0,"total":22000.0}]},{"name":"Mexallon","orders":[{"quantity":60,"price_per_unit":54.07,"total":3244.2}]},{"name":"Isogen","orders":[{"quantity":2,"price_per_unit":458.35,"total":916.7}]},{"name":"Nocxium","orders":[{"quantity":2,"price_per_unit":1191.13,"total":2382.26}]},{"name":"Zydrine","orders":[{"quantity":111,"price_per_unit":1845.41,"total":204840.51}]},{"name":"Megacyte","orders":[{"quantity":22,"price_per_unit":5932.53,"total":130515.65999999999}]}],"total":386559.33},"job_cost":{"estimated_item_value":177913.0327276708,"system_cost_index":0.0751,"facility_tax":0.002,"total":13583.126309659483,"job_cost_modifier":0.03},"total_run":400142.4563096595,"total_per_unit":400142.4563096595},"time":{"base_time":2700,"job_duration_modifier":0.15,"industry_skill_level":5,"advanced_industry_skill_level":5,"total_time_run":1560,"number_of_run_per_hour":2.3076923076923075,"number_of_run_per_day":55.38461538461538},"volume":{"per_unit_volume":25.0,"per_run_volume":25.0,"per_hour_volume":57.692307692307686,"per_day_volume":1384.6153846153845},"profit":{"markets_profit":[{"name":"Jita IV - Moon 4 - Caldari Navy Assembly Plant","profit_per_hour":-496482.5914838296,"profit_per_day":-11915582.19561191}]}}]}
